### DISCUSSION TITLE
Sharing account for personal and business work

### DISCUSSION THREAD

**Discussion by jakubjab** [community/community#64920] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I'm using the same GitHub account for personal projects and work.
I have my own Copilot for Individuals subscription, while the company has a Copilot for Business subscription.

The intention is that:
- When I'm working on personal repos, I'd like to enable "Suggestions matching public code", and I also don't mind "Allow GitHub to use my code snippets for product improvements".
- When I'm working on the company's repos, I need to follow whatever privacy rules the organization admins decide.

How can I configure GitHub Copilot in JetBrains IntelliJ to achieve that?

I think it's pretty common use case for developers to have both personal and business repos, maybe it deserves a chapter in FAQ.

A bit related to #49912, but still not clear for me.

**Comment by Akash1134** [comment, marked as answer]
Hi @JakubJablonski2-TomTom ,

We recommend that you keep Copilot for Business and Copilot for Individual accounts linked to separate GitHub accounts (dotcom user).

Depending on the [telemetry settings](https://docs.github.com/en/copilot/configuring-github-copilot/configuring-github-copilot-settings-on-githubcom) for your Copilot for Individual account (if you have the data transmission enabled), you might end up transmitting prompts or suggestions if both your business and personal accounts are associated with same GitHub ID.

### [From the Docs](https://docs.github.com/en/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot)

You can either set up a GitHub Copilot subscription for your personal account, or you can set up a GitHub Copilot for Business subscription for your organization. For more information about GitHub Copilot for Individuals, see "About GitHub Copilot for Individuals."


Thanks!

**Reply by jakubjab** [reply]
Thanks for the answer. I used to have separate accounts, but then I followed the tip in [Merging multiple personal accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) page saying "We recommend using only one personal account to manage both personal and professional repositories". I guess it's time to split them again.

**Reply by Akash1134** [reply]
Copilot for Business and Individual plans are supposed to be used in different environments and to ensure data privacy we recommend using different IDs. 🙂

**Reply by jakubjab** [reply]
Can I somehow transfer GitHub Copilot for Individuals subscription between accounts, without cancelling it and paying $100 again?

**Reply by hades200082** [reply]
This is a completely stupid policy from github.

You've recommended for years that people have just one github account for work and personal across all orgs they work with. And now you've built a tool that can't tell the difference between work and personal. 

Why can't you jsut add a dropdown to the copilot chat box to select which license to use. You could have it auto-select based on the repo's organisation too.

A completely avoidable/solvable problem that shouldn't exist.

**Reply by Triopticon** [reply]
GitHub = Microsoft --> garbage bin
It is time to find something European that actually respect people.

**Comment by innoxentwarrior** [comment]
When balancing personal and business work, using high-quality materials like centerless ground bar stock is essential for precision and efficiency. For businesses needing [Bar Chamfer Chicago](https://www.precisiongroundbars.com/centerless-ground-bar-stock/), precision-ground bars ensure smooth finishes and accuracy in manufacturing processes, supporting both personal and professional projects that demand reliability and consistency.
